pkg/init/cmd/service/prepare.go

```diff
@@ -0,0 +1,220 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"syscall"
+
+	"github.com/vishvananda/netlink"
+)
+
+const (
+	wgPath      = "/usr/bin/wg"
+	nsenterPath = "/usr/bin/nsenter-net"
+)
+
+// Note these definitions are from moby/tool/src/moby/config.go and should be kept in sync
+
+// Runtime is the type of config processed at runtime, not used to build the OCI spec
+type Runtime struct {
+	Mkdir      []string    `yaml:"mkdir" json:"mkdir,omitempty"`
```

Comment:
This is a _host_ path, I think? I only know this because we discussed it once IRL, a less well connected user might find this surprising though so it might need mentioning somewhere, probably docs in preference to e.g. a comment here (although both would be better).